patches/server/0725-Route-sign-run_command-click-events-through-normal-c.patch

```diff
@@ -13,15 +13,19 @@ the server, and prior to this patch plugins would not be able to intercept the c
 `PlayerCommandPreprocessEvent`.
 
 diff --git a/src/main/java/net/minecraft/world/level/block/entity/SignBlockEntity.java b/src/main/java/net/minecraft/world/level/block/entity/SignBlockEntity.java
-index 9b5d11ece006d7aa893360a84ba652c666517ac1..eb96018cec5adca2eb85081ba084631d712ee722 100644
+index 9b5d11ece006d7aa893360a84ba652c666517ac1..580c7a23d40ccf4add6652f6383170af0de86763 100644
 --- a/src/main/java/net/minecraft/world/level/block/entity/SignBlockEntity.java
 +++ b/src/main/java/net/minecraft/world/level/block/entity/SignBlockEntity.java
-@@ -227,7 +227,7 @@ public class SignBlockEntity extends BlockEntity implements CommandSource { // C
+@@ -226,8 +226,10 @@ public class SignBlockEntity extends BlockEntity implements CommandSource { // C
+             Style chatmodifier = ichatbasecomponent.getStyle();
              ClickEvent chatclickable = chatmodifier.getClickEvent();
  
-             if (chatclickable != null && chatclickable.getAction() == ClickEvent.Action.RUN_COMMAND) {
+-            if (chatclickable != null && chatclickable.getAction() == ClickEvent.Action.RUN_COMMAND) {
 -                player.getServer().getCommands().performCommand(this.createCommandSourceStack(player), chatclickable.getValue());
-+                player.connection.chat(chatclickable.getValue(), false); // Paper - route through standard chat/command logic
++            // Paper start - route through standard chat/command logic
++            if (chatclickable != null && chatclickable.getAction() == ClickEvent.Action.RUN_COMMAND && chatclickable.getValue().startsWith("/")) {
++                player.connection.chat(chatclickable.getValue(), false);
```

Comment:
I believe this needs an extra `/` added for WorldEdit and whatnot. Please verify whether this works on commands such as `/wand` (i.e. the player types `//wand`) first.